app/views/cdo/manage/cdo.njk

```diff
@@ -27,34 +28,37 @@
 
 {% block content %}
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-three-quarters">
     <form class="form" method="post">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-one-half">
-          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">Dog {{ model.details.dogIndex }}</h1>
-        </div>
-
-        <div class="govuk-grid-column-one-half defra-text-align-right">
-          <div class="govuk-button-group defra-align-right-simple govuk-!-margin-bottom-1">
-            <a href="/cdo/view/dog-details/{{ model.details.dogIndex }}{{ model.srcHashParam }}&force=true" role="button" draggable="false" class="govuk-button govuk-button--secondary" data-module="govuk-button">
-              Dog record
-            </a>
-            <a href="/cdo/view/owner-details/{{ model.details.personReference }}{{ model.srcHashParam }}" role="button" draggable="false" class="govuk-button govuk-button--secondary" data-module="govuk-button">
-              Owner record
-            </a>
+      <div class="govuk-grid-column-three-quarters govuk-!-margin-bottom-2">
```

Comment:
Does this do what I need for the changes to 'View Dog Record' (481992). If so, great but I've had to clarify with Nick exactly how he wants it to style when mobile-sized. And it's not standard GDS